pkg/autoupdate/tuf/autoupdate.go

```diff
@@ -46,14 +48,20 @@ type ReleaseFileCustomMetadata struct {
 }
 
 type librarian interface {
+	IsInstallVersion(binary autoupdatableBinary, targetFilename string) bool
 	Available(binary autoupdatableBinary, targetFilename string) bool
 	AddToLibrary(binary autoupdatableBinary, targetFilename string, targetMetadata data.TargetFileMeta) error
-	TidyLibrary()
+	TidyLibrary(binary autoupdatableBinary, currentVersion string)
```

Comment:
Why change this? I imagine most callers would just want to tidy everything.

Reply:
Ah, reading more code, I think it's so that we shift the responsibility of finding the running version to the caller.